olclient/bots.py

```diff
@@ -42,11 +42,11 @@ def __init__(self, ol=None, dry_run=True, limit=1, job_name=__name__) -> None:
             '-d',
             '--dry-run',
             type=self._str2bool,
-            default=True,
             help='Execute the script without performing edits on external data.',
         )
         self.args = self.parser.parse_args()
-        self.dry_run = getattr(self.args, 'dry-run', None) or dry_run
+        dry_run_from_args = getattr(self.args, 'dry_run', None)
+        self.dry_run = dry_run_from_args if dry_run_from_args is not None else dry_run
```

Comment:
```suggestion
        self.dry_run = dry_run_from_args or dry_run
```

Reply:
`dry_run_from_args` should be given higher priority for a cli bot as far as I know. If `dry_run_from_args` is passed as false, `dry_run` will anyways be true since its default is set from the `__init__` method. 

Then `False or True` will evaluate to `True`, even if `dry_run_from_args` is explicitly passed as `False`.